addOns/graphql/src/main/java/org/zaproxy/addon/graphql/GraphQlFingerprinter.java

```diff
@@ -436,6 +439,16 @@ private boolean checkHasuraEngine() {
         return errorContains("missing selection set for \"__Schema\"");
     }
 
+    private boolean checkHotchocolateEngine() {
+        sendQuery("queryy  {__typename}");
```

Comment:
Is `queryy` spelling mistake deliberate?

Reply:
(another instance below)

Reply:
Hahaha ya it's actually intentional. That caught me when I started too.